lib/src/main/java/dev/openfeature/javasdk/FeatureProvider.java

```diff
@@ -5,4 +5,5 @@ public interface FeatureProvider {
     ProviderEvaluation<Boolean> getBooleanEvaluation(String key, Boolean defaultValue, EvaluationContext ctx, FlagEvaluationOptions options);
     ProviderEvaluation<String> getStringEvaluation(String key, String defaultValue, EvaluationContext ctx, FlagEvaluationOptions options);
     ProviderEvaluation<Integer> getIntegerEvaluation(String key, Integer defaultValue, EvaluationContext ctx, FlagEvaluationOptions options);
+    // what about getObjectEvaluation? Have you considered ObjectMapper? https://fasterxml.github.io/jackson-databind/javadoc/2.7/com/fasterxml/jackson/databind/ObjectMapper.html
```

Comment:
I haven't done this yet. I haven't figured out how I want to do it, so I've just left it off. https://github.com/open-feature/java-sdk/issues/7